docx/oxml/section.py

```diff
@@ -119,6 +125,18 @@ def gutter(self, value):
         pgMar = self.get_or_add_pgMar()
         pgMar.gutter = value
 
+    def has_header_reference(self, header_type):
+        """ takes a ```w:type``` of default, even, or odd and returns |True|
+        if the corresponding ```w:headerReference``` exists in the sectPr
+        else |False|
+        """
+        HR_TYPE = '{%s}type' % NS.WML_MAIN
+        for ref in self.headerReference_lst:
+            if ref.attrib.get(HR_TYPE) == header_type:
+                return True
+
+        return False
+
```

Comment:
The get_headerReference_of_type() method you write will replace this one. The True or False will just be whether it returns None or not. It will be reused in a get_or_add_headerReference_of_type() method I expect we'll need just a bit later.

Note the element name is used just as it appears in the XML, and is not converted to snake-case.